src/Microsoft.TemplateEngine.TemplateLocalizer.Core/Utilities/Rune.cs

```diff
@@ -0,0 +1,542 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.Text
+{
+    // netstandard2.0 compatible implementation of System.Text.Rune.
```

Comment:
do we need to support netstandard?

Reply:
We already shipped 6.0.100 version of this package to NuGet with .NET Standard 2.0. We can still change it, but we should be aware of the consequences (breaking people that are targeting netstandard2.0). This is also being consumed by our Tasks package, so its target framework needs to be updated as well.

Reply:
Supporting these files also has an associated cost. If supporting standard is not a must (not sure about VS build), we might want to revise supporting it.

Reply:
After discussing internally, we decided to continue targeting .NET Standard 2.0.